polaris/polaris/utils.py

```diff
@@ -242,11 +242,12 @@ def create_transaction_envelope(transaction, source_account) -> TransactionEnvel
         transaction.asset.significant_decimals,
     )
     memo = make_memo(transaction.memo, transaction.memo_type)
-    base_fee = settings.HORIZON_SERVER.fetch_base_fee()
     builder = TransactionBuilder(
         source_account=source_account,
         network_passphrase=settings.STELLAR_NETWORK_PASSPHRASE,
-        base_fee=base_fee,
+        # only one operation, so base_fee will be multipled by 1
+        base_fee=settings.MAX_TRANSACTION_FEE_STROOPS
```

Comment:
Quick question, if `settings.MAX_TRANSACTION_FEE_STROOPS` = 500 and `settings.HORIZON_SERVER.fetch_base_fee()` = 100 
will base fee be 100 or 500?
```
...
base_fee=settings.MAX_TRANSACTION_FEE_STROOPS
or settings.HORIZON_SERVER.fetch_base_fee(),
```

Reply:
It will be 500 because the 2nd piece of the `or` statement is only evaluated if the 1st is falsey